nncf/torch/compression_method_api.py

```diff
@@ -112,15 +113,28 @@ def load_state(self, state: Dict[str, object]) -> None:
 
         :param state: Output of `get_state()` method.
         """
-        self.scheduler.load_state(state)
+        if self.scheduler is not None:
+            self.scheduler.load_state(state['scheduler'])
+        if 'compression_level' in state:
+            if 'compression_stage' not in state:
+                compression_level = state['compression_level']
+                state['compression_stage'] = CompressionLevel.map_legacy_level_to_stage()[compression_level]
+            else:
+                nncf_logger.warning('Both CompressionStage and (legacy) CompressionLevel attributes '
+                                    'are specified in the checkpoint. Proceeding with the value stored '
+                                    'in CompressionStage')
+        if self.compression_stage() != state['compression_stage']:
```

Comment:
@ljaljushkin Just adding a check of whether `CompressionStage` has the same value in the controller and in the checkpoint. This logic can probably be extended when the full controller state saving/loading functionality will be implemented.